docs/api/io/users.rst

```diff
@@ -1 +1,2 @@
 .. automodule:: tenable.io.users
```

Comment:
Update doc as per this PR: 
https://github.com/dhvanilshah-tenable/pyTenable/pull/18/files#

1. Remove docs/api/io/users.rst 
2. Add docs/api/io/v3/vm/scanners.rst 
3. Remove docs/api/io/v3/index.rst